selftests/unit/test_xunit.py

```diff
@@ -48,7 +48,7 @@ def test(self):
         self.test1._Test__status = 'PASS'
         self.test1.time_elapsed = 1.23
         self.junit = os.path.abspath(os.path.join(os.path.dirname(__file__),
-                                     os.path.pardir, ".data", 'junit-4.xsd'))
+                                                  os.path.pardir, ".data", 'junit-4.xsd'))
```

Comment:
the same here, you could probably assign the `rel_path` first and then use `abspath` on that variable...